test/e2e/snaps/test-snap-managestate.spec.js

```diff
@@ -70,9 +70,14 @@ describe('Test Snap manageState', function () {
         });
 
         // fill and click send inputs on test snap page
-        windowHandles = await driver.waitUntilXWindowHandles(2, 1000, 10000);
+        windowHandles = await driver.waitUntilXWindowHandles(1, 1000, 10000);
         await driver.switchToWindowWithTitle('Test Snaps', windowHandles);
+        await driver.delay(1000);
```

Comment:
Were lower numbers tried and where does the number 1000 come from? 1s is relatively high (they add up) and if lower numbers still reliably pass it would be great (the same applies to other delays in these suites).

Either way some seconds increase in delay is still better than false negatives in CI so can be addressed separately.